src/Discord.Net.Rest/ClientHelper.cs

```diff
@@ -152,5 +152,11 @@ internal static class ClientHelper
             var models = await client.ApiClient.GetVoiceRegionsAsync(options).ConfigureAwait(false);
             return models.Select(x => RestVoiceRegion.Create(client, x)).FirstOrDefault(x => x.Id == id);
         }
+
+        public static async Task<IReadOnlyCollection<RestRelationship>> GetRelationshipsAsync(BaseDiscordClient client)
+        {
+            var models = await client.ApiClient.GetRelationshipsAsync().ConfigureAwait(false);
+            return models.Select(r => RestRelationship.Create(client, r)).ToImmutableArray();
```

Comment:
AFAIK, `ToReadOnlyCollection()` is preferred when you have a collection, instead of `ToImmutableArray()`.